vueui/src/components/view/mcomputer/generic/ntsl/terminal.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <vui-button :params="{refresh: 1}">Refresh</vui-button> <vui-button :params="{closefile: 1}">Exit</vui-button>
+    <h2>{{running_name}}</h2>
+    <div class="item">
+      <div class="itemContent" style="width: 100%; font-family: monospace; line-height:8pt; font-size:8pt; text-align:center;">
+        <span id="ntsl_terminalbody" v-html="terminal"/><br>
+        <input id="ntsl_commandline" v-on:keyup.enter="commandLine()">
+      </div>
+    </div>
+  </div>
+</template>
+<!-- Can't scope due to custom HTML injection. Shoot me if you use ntsl_terminalbody as an ID for something else -->
+<style lang="scss">
```

Comment:
Please use scoped stling and less clunky selectors. Also do not have multiple style tags and styles go after script tag.

Reply:
This is required as manually setting inner HTML doesn't add the scoped data to tags inserted